src/core/xccl_team.c

```diff
@@ -98,15 +83,15 @@ xccl_status_t xccl_team_create_test(xccl_team_t *team)
     /* TODO: check if some teams are never used after selection and clean them up */
     return XCCL_OK;
 }
-xccl_status_t xccl_team_destroy(xccl_team_t *team)
+
+void xccl_team_destroy(xccl_team_t *team)
 {
-    XCCL_CHECK_TEAM(team);
```

Comment:
do you think XCCL_CHECK_TEAM is not necessary? I thought it can save some debugging time if user forgets to wait for team_create_post completion .

Reply:
Removed because all destroy/free functions return void and CHECK_TEAM has return.  Maybe for team_destroy just check for status and print message without using this macro?